[Http-Requests 2018-08-24 15:47:16,513 HTTP_requests.py HTTP_requests  INFO] ===》 获取接口地址成功：ht[damao 2018-08-24 15:47:17,319 test_my_demo.py test_my_demo  INFO] ===》 请求体参数：params:{'c': 1}
[damao 2018-08-24 15[Http-Requests 2018-08-24 15:47:17,336 HTTP_requests.py HTTP_requests  INFO] ===》 获取接口地址成功：http://httpbin.org/
[Http-Requests 2018-08-24 15:47:17,932 HTTP_requests.py HTTP_requests  INFO] ===》 get请求发送成功！
ate', 'Accept': '*/*', 'Connection': 'keep-alive'}
[damao 2018-08-2[python 2018-08-24 15:47:17,941 run.py run  INFO] ===》 剪切失败：Destination path '.\logs\zip_log\auto_test_2018-08-18-02-12-47.log' already exists
[python 2018-08-24 15:47:17,941 run.py run  INFO] ===》 剪切失败：Destination path '.\logs\zip_log\auto_test_2018-08-18-02-14-21.log' already exists
[python 2018-08-24 15:47:17,941 run.py run  INFO] ===》 剪切失败：Destination path '.\logs\zip_log\auto_test_2018-08-18-02-19-01.log' already exists
[python 2018-08-24 15:47:17,942 run.py run  INFO] ===》 剪切失败：Destination path '.\logs\zip_log\auto_test_2018-08-18-02-19-02.log' already exists
[python 2018-08-24 15:47:17,951 run.py run  INFO] ===》 4条剪切成功！
[python 2018-08-24 15:47:17,953 run.py run  INFO] ===》 5条剪切成功！
[python 2018-08-24 15:47:17,956 run.py run  INFO] ===》 6条剪切成功！
[python 2018-08-24 15:47:17,958 run.py run  INFO] ===》 7条剪切成功！
[python 2018-08-24 15:47:17,961 run.py run  INFO] ===》 8条剪切成功！
[python 2018-08-24 15:47:17,966 run.py run  INFO] ===》 剪切失败：[WinError 32] 另一个程序正在使用此文件，进程无法访问。: '.\\logs\\log_file\\auto_test_2018-08-24-15-47-16.log'
